pandas/tests/series/indexing/test_setitem.py

```diff
@@ -126,6 +127,13 @@ def test_setitem_boolean_different_order(self, string_series):
 
         tm.assert_series_equal(copy, expected)
 
+    def test_setitem_boolean_ea_type(self):
+        # GH: 26468
+        s = Series(array([5, 6, 7, 8], dtype="Int64"))
+        s[s > 6] = Series(range(4), dtype="Int64")
```

Comment:
generally looks good, couple of nitpicks: can you `import array as pd_array`, name this `ser` instead of `s` (doesnt  have to be `ser`, the important thing is to avoid 1-letter variable names), instead of "ea_type" use "Int64_values" (or better yet, parametrize over nullable int dtypes)

Reply:
Done, thanks for the tips